configuration/default-applications.md

```diff
@@ -0,0 +1,125 @@
+
+	
+	[Default Applications]
+	x-scheme-handler/unknown=qvm-open-in-dvm.desktop
+	x-scheme-handler/about=qvm-open-in-dvm.desktop
+	x-scheme-handler/http=qvm-open-in-dvm.desktop
+	x-scheme-handler/https=qvm-open-in-dvm.desktop
+	text/html=qvm-open-in-dvm.desktop
+	text/xml=qvm-open-in-dvm.desktop
+	image/gif=qvm-open-in-dvm.desktop
+	image/jpeg=qvm-open-in-dvm.desktop
+	image/png=qvm-open-in-dvm.desktop
+	application/xhtml+xml=qvm-open-in-dvm.desktop
+	application/xml=qvm-open-in-dvm.desktop
+	application/vnd.mozilla.xul+xml=qvm-open-in-dvm.desktop
+	application/rss+xml=qvm-open-in-dvm.desktop
+	application/rdf+xml=qvm-open-in-dvm.desktop
+	application/msword=qvm-open-in-dvm.desktop
+	application/vnd.oasis.opendocument.spreadsheet=qvm-open-in-dvm.desktop
+	application/vnd.oasis.opendocument.text=qvm-open-in-dvm.desktop
+	application/pdf=qvm-open-in-dvm.desktop
+
+(You can save this file in `/usr/share/applications/` instead; but if your DispVM is built from that template it, too, will open these kind of files with `qvm-open-in-dvm`, which you may not want.)
+
+Still in the VM(s) you want to run `qvm-open-in-dvm` from: You need to create a corresponding `.desktop` file and save it in `/usr/share/applications/` or `~/.local/share/applications/`, depending on what you prefer. Every MIME type assigned to this application in `mimeapps.list` must be listed in the `.desktop` file, but fortunately wildcards can be used for subtypes. Example:
+
+	[Desktop Entry]
+	Encoding=UTF-8
+	Name=QvmOpenInDVM
+	Exec=qvm-open-in-dvm %u
+	Terminal=false
+	X-MultipleArgs=false
+	Type=Application
+	Categories=Qubes
+	MimeType=x-scheme-handler/*;text/*;application/*;image/*;
+
+Now, whenever a file (or URL) of these MIME types is opened, the will be opened in a new DispVM (unless they are explicitly opened with another application, of course). You can open any such file in a DispVM by executing `xdg-open filename`.
+
+ `qvm-open-in-dvm filename` tells dom0 *that* the file be opened in a new DispVM; `mimeapps.list` on the DispVM determines *how* the file is handled internally by the DispVM. To have these files open in your preferred application *on the DispVM* set up a `mimeapps.list` like you would in any other VM.
+
+
+## How to open URLs with Tor Browser Bundle by default.
+
+Assuming that Tor Browser Bundle (TBB) is installed in its standard location, create `start-tor-browser.desktop` in `~/.local/share/applications/` with the following content:
+
+	[Desktop Entry]
+	Type=Application
+	Name=Tor Browser
+	GenericName=Web Browser
+	Comment=Tor Browser is +1 for privacy and -1 for mass surveillance
+	Categories=Network;WebBrowser;Security;
+	Exec=/home/user/.local/share/torbrowser/tbb/x86_64/tor-browser_en-US/Browser/start-tor-browser
```

Comment:
Actually it's `%k` by default. I have removed the Tor Browser part and will add it to a new how-to with some related tweaks.